docs/ml-features.md

```diff
@@ -654,7 +654,11 @@ for expanded in polyDF.select("polyFeatures").take(3):
 `StringIndexer` encodes a string column of labels to a column of label indices.
 The indices are in `[0, numLabels)`, ordered by label frequencies.
 So the most frequent label gets index `0`.
-If the input column is numeric, we cast it to string and index the string values.
+If the input column is numeric, we cast it to string and index the string 
+values. When following pipeline components such as `Estimator` or 
```

Comment:
This may be nit-picky or just my brain, but I misread this sentence a few times as "when you are following ... you use this string indexed ..." which then isn't parseable. Maybe, "When downstream pipeline components ... make use of ..."

"set input column" -> "set the input column"
"string indexed" -> "string-indexed"
you can set with -> you can set the input column with
drop parens? it's a useful standalone sentence